deploy/charts/trust-manager/templates/deployment.yaml

```diff
@@ -16,6 +16,8 @@ spec:
         app: {{ include "trust-manager.name" . }}
     spec:
       serviceAccountName: {{ include "trust-manager.name" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
```

Comment:
I think the existing behavior should not change. Better to omit `securityContext` by default.